modules/installation-requirements-user-infra-ibm-z-kvm.adoc

```diff
@@ -66,7 +66,12 @@ The {op-system-base} KVM host in your environment must meet the following requir
 You can install {product-title} version {product-version} on the following IBM hardware:
 
 * IBM z16 (all models), IBM z15 (all models), IBM z14 (all models), IBM z13, and IBM z13s
-* LinuxONE, any version
+* IBM LinuxONE Emperor 4, IBM LinuxONE III (all models), IBM LinuxONE Emperor II, IBM LinuxONE Rockhopper II, IBM LinuxONE Emperor, and IBM LinuxONE Rockhopper
+
+[NOTE]
+====
+Support for {op-system} functionality for IBM z13 all models, LinuxONE Emperor, and LinuxONE Rockhopper is deprecated. While these hardware models remain fully supported in {product-title} 4.12, Red Hat recommends that you use later hardware models.
```

Comment:
IBM Style Guide recommends not using _while_ to mean _although_ or _though_, as in this context. 
Maybe:
_Even thought these hardware models..._
_These hardware models remain fully supported in {product-title} 4.12. However, Red Hat recommends...